app/src/main/java/de/xikolo/controllers/cast/CastActivity.java

```diff
@@ -24,14 +27,15 @@ public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
         getMenuInflater().inflate(R.menu.cast, menu);
         CastButtonFactory.setUpMediaRouteButton(this, menu, R.id.media_route_menu_item);
+        getMenuInflater().inflate(R.menu.helpdesk, menu);
```

Comment:
The overflow menu in `CastActivity` has the wrong theme applied to it.
<img width="200px" src="https://user-images.githubusercontent.com/26904189/68666686-a6a57e00-0544-11ea-9d87-6b88704e7a26.jpg" />